runtime/doc/nvim_clipboard.txt

```diff
@@ -29,4 +35,77 @@ following option:
 See 'clipboard' for details and more options.
 
 ==============================================================================
+2. X11 selection mechanism				    *nvim-clipboard-x11*
```

Comment:
:+1: to moving this, but the tag should remain `*x11-selection*`. I don't know why we're prefixing a bunch of tags with `nvim-`, I think we need to reconsider that practice...